src/docker-in-docker/install.sh

```diff
@@ -332,13 +333,64 @@ fi
 
 usermod -aG docker ${USERNAME}
 
+# Function to fetch the latest version of the plugin
+get_latest_version() {
+    repo_url=$1
+    curl -s "$repo_url/latest" | jq -r '.tag_name'
```

Comment:
![image](https://github.com/devcontainers/features/assets/24955913/44631ffb-3ae8-4683-9171-3ec665e99fc4)

Doesn't work for me!